spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -0,0 +1,63 @@
+import { When, Then } from "cypress-cucumber-preprocessor/steps";
+
+const elements = {
+  "cookie consent": '[name="cookies_form[analytics_consent]"]',
+  "cookie banner": ".js-cookie-banner",
+  "create admin button": "[data-test=create-admin-button]",
+  main: "main",
+  name: "input[name*=full_name]",
+  email: "input[name*=email]",
+  "notification banner": "[data-test=notification-banner]",
+  "edit admin link": "[data-test=edit-admin-link]",
+  "delete button": "[data-test=delete-button]",
```

Comment:
Same, most of these probably shouldn't be here

Reply:
Yeah, I got rid of them. I kept the general interaction parts, cause I think they will be pretty useful for translating the cypress tests we have to cucumber though.